app/jobs/mesh_validate_mailbox_job.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class MESHValidateMailboxJob < ApplicationJob
+  queue_as :default
```

Comment:
We've got a number of different kinds of jobs now that all go in the same queue, do we need to think about prioritisation here? For example, I imagine we wouldn't want this job to get stuck behind loads of consent reminders.

Reply:
Ya. Sounds reasonable to create a queue for MESH

Reply:
Added the queue